src/main/java/org/humancellatlas/ingest/submission/SubmissionEnvelope.java

```diff
@@ -127,4 +128,21 @@ public boolean isOpen() {
         List<SubmissionState> states = Arrays.asList(SubmissionState.values());
         return states.indexOf(this.getSubmissionState()) < states.indexOf(SubmissionState.SUBMITTED);
     }
+
+    @JsonIgnore()
+    public boolean isInEditableState() {
```

Comment:
could simplify name to `isEditable`

Reply:
or perhaps something like `isStateInTransition` ?

Reply:
I prefer isEditable since it describes what we actually want to check better.